person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] Your `of_age?` method should be a private method. Kindly make it so. You can move this method to the bottom of the class and under a `private` keyword